docs/src/ex/0_Introduction.md

```diff
@@ -0,0 +1,178 @@
+data[1:5]
+```
+
+After flipping all our coins, we want to set a prior belief about what we think the distribution of coinflips look like. In our case, we are going to choose a common prior distribution called the [Beta](https://en.wikipedia.org/wiki/Beta_distribution) distribution. We will allow this distribution to change as we let our model see more evidence of coin flips.
+
+```julia
+# our prior belief about the probability of heads in a coin toss
+prior_belief = Beta(1, 1);
+```
+
+With our priors set and our data at hand, we can finally run our simple posterior update model.
+
+This is a fairly simple process. We expose one additional coin flip to our model every iteratior, such that the first run only sees the first coin flip, while the last iteration sees all the coin flips. Then, we set the `updated_belief` variable to an updated version of the original Beta distribution after accounting for the new proportion of heads and tails.
+
+```julia
+# this is required for plotting only
+x = range(0, stop = 1, length = 100)
+
+# make an animation
+animation = @animate for (i, N) in enumerate(Ns)
+
+    # count the number of heads and tails
+    heads = sum(data[1:i-1])
+    tails = N - heads
+
+    # update our prior belief in closed form (this is possible because we use a conjugate prior)
+    updated_belief = Beta(prior_belief.α + heads, prior_belief.β + tails)
+
+    # plotting
+    plot(x, pdf.(Ref(updated_belief), x),
+        size = (500, 250),
+        title = "Updated belief after $N observations",
+        xlabel = "probability of heads",
+        ylabel = "",
+        legend = nothing,
+        xlim = (0,1),
+        fill=0, α=0.3, w=3)
+    vline!([p_true])
+end;
+```
+
+![animation](https://user-images.githubusercontent.com/7974003/44995702-37c1b200-af9c-11e8-8b26-c88a528956af.gif)
+
+The animation above shows that with increasing evidence our belief about the probability of heads in a coin flip slowly adjusts towards the true value. The orange line in the animation represents the true probability of seeing heads on a single coin flip, while the mode of the distribution shows what the model believes the probability of a heads is given the evidence it has seen.
+
+## Coin Flipping with Turing
+
+In the previous example, we used the fact that our prior distribution is a [conjugate prior](https://en.wikipedia.org/wiki/Conjugate_prior). Note that a closed-form expression (the `updated_belief` expression) for the posterior is not accessible in general and usually does not exist for more interesting models.
+
+We are now going to move away from the closed-form expression above and specify the same model using Turing*. To do so, we will first need to import `Turing`, `MCMCChain`, `Distributions`, and `StatPlots`. `MCMChain` is a library built by the Turing team to help summarize Markov Chain Monte Carlo (MCMC) simulations, as well as a variety of utility functions for diagnostics and visualizations.
```

Comment:
*Turing*

Reply:
Good catch. I've bolded it, the new paragraph is:

> ... and specify the same model using **Turing**. ...